sass/MLFacet/_MLFacetValues.scss

```diff
@@ -0,0 +1,47 @@
+@import '../Variables';
+
+.coveo-ml-facet-values {
+  margin: 15px 0;
+  padding: 0;
+}
+
+.coveo-ml-facet-value {
+  list-style: none;
+  margin-bottom: 6px;
+
+  &.coveo-selected {
+    .coveo-checkbox-span-label,
+    .coveo-checkbox-span-label-suffix {
+      font-weight: bold;
+    }
+  }
+
+  .coveo-checkbox-label {
+    width: 100%;
+  }
+
+  .coveo-checkbox-span-label {
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+  }
+
+  .coveo-checkbox-span-label-suffix {
+    margin-left: 8px;
+    color: $color-light-grey;
+  }
+
+  .coveo-checkbox-button:focus {
+    & ~ .coveo-checkbox-span-label,
```

Comment:
Could we do without the specificity of this selector ?

By maybe adding a longer name to the css class name ? 

It's generally easier for clients/integration to overrides some styles if needed if the css rules are flat, but with a longer name to differentiate them.

Reply:
So add/remove a class with js?